modules/standard/IO.chpl

```diff
@@ -7287,13 +7344,16 @@ proc fileWriter.writeBytes(b: bytes, size = b.size) throws {
    bytes in length, and ``numBytes=17``, only 16 bytes will be written.
 
    .. warning::
-      This method provides no protection against attempting to access invalid memory
+      This method provides no protection against attempting to access invalid
+      memory
 
    :arg ptr: a :class:`~CTypes.c_ptr` to some valid memory
    :arg numBytes: the number of bytes to write
 
-   :throws UnexpectedEofError: Thrown if the write operation exceeds the ``fileWriter``'s specified range.
-   :throws SystemError: Thrown if an error occurred while writing to the ``fileWriter``.
+   :throws UnexpectedEofError: Thrown if the write operation exceeds the
+                               ``fileWriter``'s specified range.
```

Comment:
Should we say "region" instead of "range" to match the argument name in other places?